src/conf_mode/service_pppoe-server.py

```diff
@@ -43,8 +45,36 @@ def get_config(config=None):
 
     # retrieve common dictionary keys
     pppoe = get_accel_dict(conf, base, pppoe_chap_secrets)
+
+    # We have gathered the dict representation of the CLI, but there are default
+    # options which we need to update into the dictionary retrived.
+    default_values = defaults(base)
+    pppoe = dict_merge(default_values, pppoe)
```

Comment:
`get_accel_dict()` already takes care of handling the default values from CLI.

If you found a corner case that is missing in `get_accel_dict()` please implement it. If you find missmatching CLI definitions that are handled in `get_accel_dict()` but are differnt in the pppoe-server implementation we should rather migrate the CLI to have a consistent look and feel for the user.

Reply:
Fixed check in `verify_accel_ppp_base_service`